newa/cli.py

```diff
@@ -205,62 +206,193 @@ def cmd_event(ctx: CLIContext, errata_ids: list[str]) -> None:
 def cmd_jira(ctx: CLIContext, issue_config: str) -> None:
     ctx.enter_command('jira')
 
-    # this is here temporarily so we generate fake Jira issue IDs
-    jira_id_gen = itertools.count(start=1)
+    jira_url = ctx.settings.jira_url
+    if not jira_url:
+        raise Exception('Jira URL is not configured!')
+
+    jira_token = ctx.settings.jira_token
+    if not jira_token:
+        raise Exception('Jira URL is not configured!')
 
     for erratum_job in ctx.load_erratum_jobs('event-'):
+
         # read Jira issue configuration
         config = ErratumConfig.from_yaml_file(Path(os.path.expandvars(issue_config)))
 
-        # TODO: record created/existing issues. Instead of `Any`, maybe something
-        # from the Jira library would be stored. Or just a Jira ticket ID.
-        known_issues: dict[str, Any] = {}
+        jira = IssueHandler(jira_url, jira_token, config.project, config.transitions)
```

Comment:
Followup on a conversation from issue#22, do you think it makes sense to read project settings from a config file, envvar, jira subcommand `--project` option etc? I did something similar in
https://github.com/RedHatQE/newa/pull/27/files#diff-fbfe615aa583117b7a57302f5da6ff45c1eda26a6f4b2406b68a09ffe1bd698aR383
but TBH I am not sure which settings should have priority. Or would you rather "enforce" this kind of settings for the issue config file?

Reply:
I considered it before making this change and here's the situation a bit more complicate because of `transitions` - it would be hard (or at least too verbose) to express resolution transitions (that is how jira calls it) on a command line or via envvar. So that part would end up in component config anyway and hence moving project out of it won't help too much. 

I'd propose to keep it all there (in a component config) for now and later we try to see what resolution transitions we will be working. If there is some reasonable default for most of the projects then we can assume that default and then we would only need `transitions` setting for special cases. In that case having `project` configurable either by an option of by envvar would make more sense.

Reply:
Ack